locales/en/app.ftl

```diff
@@ -29,6 +29,7 @@ error-not-subscribed = This email address is not subscribed to {-product-name}.
 error-hibp-throttled = Too many connections to {-brand-HIBP}.
 error-hibp-connect = Error connecting to {-brand-HIBP}.
 error-hibp-load-breaches = Could not load breaches.
+error-must-be-signed-in = You must be signed in to your {-brand-fxa}.
```

Comment:
@betsymi right now I'm just including the first sentence from your copy, as it's the most applicable to everywhere this error message shows up. Is that okay?

Reply:
I'm good with it.

Reply:
@groovecoder: Do we have an issue for adding the second part of the error messsage? I didn't see one in my quick glance at the issues, but wanted to confirm before filing.